.github/workflows/build.yml

```diff
@@ -73,37 +73,37 @@ jobs:
       if: matrix.os == 'windows-latest'
       env:
         TRACE_NODE_API_HOST: 1
-      run: dotnet test -f net472 --configuration Release --logger trx --results-directory "test-netfx47-node${{ matrix.node-version }}"
+      run: >
+        dotnet test -f net472
+        --configuration ${{ matrix.configuration }}
+        --logger trx
+        --results-directory "test-netfx47-node${{ matrix.node-version }}-${{ matrix.configuration }}"
 
     - name: Test .NET 6
       env:
         TRACE_NODE_API_HOST: 1
-      run: dotnet test -f net6.0 --configuration Release --logger trx --results-directory "test-dotnet6-node${{ matrix.node-version }}"
+      run: >
+        dotnet test -f net6.0
+        --configuration ${{ matrix.configuration }}
+        --logger trx
+        --results-directory "test-dotnet6-node${{ matrix.node-version }}-${{ matrix.configuration }}"
 
     - name: Test .NET 7
       env:
         TRACE_NODE_API_HOST: 1
-      run: dotnet test -f net7.0 --configuration Release --logger trx --results-directory "test-dotnet7-node${{ matrix.node-version }}"
+      run: >
+        dotnet test -f net7.0
+        --configuration ${{ matrix.configuration }}
+        --logger trx
+        --results-directory "test-dotnet7-node${{ matrix.node-version }}-${{ matrix.configuration }}"
 
     - name: Upload test logs
       uses: actions/upload-artifact@v3
       with:
-        name: test-logs-${{ runner.os }}-node${{ matrix.node-version }}
-        path: out/obj/Release/**/*.log
+        name: test-logs-${{ matrix.os }}-node${{ matrix.node-version }}-${{ matrix.configuration }}
+        path: out/obj/${{ matrix.configuration }}/**/*.log
       if: ${{ always() }}
```

Comment:
Currently if tests for one of the target frameworks fails, then the following test steps do not execute. I wonder if there is a condition such that we can run tests on successful build, but regardless of success or failure of earlier test steps?

Reply:
Agree, I was looking how to address it too. I am going to augment the PR if I find the fix.
(I found a way to test it in my fork and the code is not ready yet.)

Reply:
The latest iteration addresses it.